examples/apt/test_linux_amd64.yaml

```diff
@@ -0,0 +1,23 @@
+schemaVersion: "2.0.0"
+
+commandTests:
+  - name: "echo hello"
+    command: "/bin/bash"
+    args: ["-c", "echo hello world!"]
+    expectedOutput: ["hello world!"]
+  - name: "apt list --installed"
+    command: "apt"
+    args: ["list", "--installed"]
+    expectedOutput:
+      - Listing\.\.\.
```

Comment:
nit: is there a yaml syntax that allows this to be less escape-y ?

Reply:
this bugs me too. container-structure-test doesn't have match this exactly option. everything has to be a regex, so the escapes are side effect of regex